usecases/user/star-message.md

```diff
@@ -0,0 +1,87 @@
+<table>
+    <tr>
+        <td>
+            <strong>Name</strong>
+        </td>
+        <td>
+            star-message
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Actor</strong>
+        </td>
+        <td>
+            User
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Description</strong>            
+        </td>
+        <td>
+            Actor stars a message
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Pre-condition</strong>
+        </td>
+        <td>
+            Actor is 
+            <a href="login.md">
+            logged in
+            </a> <br>Actor has <a href="open-cat">opened</a> a chat
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Scenario</strong>
+        </td>
+        <td>
+            <ul>
+                <li>1|
+                    Actor selects a message in the opened chat
+                </li>
+                <li>
+                    2| System displays options to interact with the highlighted message
+                </li>
+                <li>
+                    3| Actor selects the star button
+                </li>
+                <li>
+                    4| System displays that the message is now starred
+                </li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Result</strong>
+        </td>
+        <td>
+            Actor has starred a message
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>Exceptions</strong>
+        </td>
+        <td>
+            <ul>
+                <li>
+                    4| User has deleted the message
+                </li>
+                <li>
+                    4| User has deleted chat history
```

Comment:
maybe add precondition, that the message which should be starred should be available.
Then this exception block would not be necessary